Tournament/Warlock_TM/map.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "WarlockTM",
+	"name": "Warlock_TM",
```

Comment:
You should **never** put underscores in a map name. Folders shouldn't have spaces they should have underscores, while for map names it should be the complete opposite.